services/self-service/src/main/java/com/epam/dlab/backendapi/dao/EndpointDAOImpl.java

```diff
@@ -65,11 +74,19 @@ public void remove(String name) {
 		deleteOne(ENDPOINTS_COLLECTION, endpointCondition(name));
 	}
 
+	/**
+	 * According the functional requirement, the EndPoint name should be case-insensitive.
+	 * A New method use the regex interface of the MongoDB driver
```

Comment:
There is no need to mention "new method" because we don't have an old one)
I assume to delete this line at all.

Reply:
The comment has been deleted